ArduCopter/motors.cpp

```diff
@@ -51,15 +51,6 @@ void Copter::arm_motors_check()
             }
         }
 
-        // arm the motors and configure for flight
-        if (arming_counter == AUTO_TRIM_DELAY && motors->armed() && flightmode->mode_number() == Mode::Number::STABILIZE) {
```

Comment:
`AUTO_TRIM_DELAY` define can be removed too.

Reply:
Done